src/pages/RadicalCollaboration/Share/index.js

```diff
@@ -130,6 +133,7 @@ const StarMediaSection = () => {
           image='/images/medium.png'
           title='Medium logo'
           cardContent='Mention or write about us on Medium'
+          link='https://medium.com/new-story'
```

Comment:
Should the link open in a new tab?

Reply:
That I'm not sure; I'd have to check with others as it wasn't specified in the issue.

Reply:
I think the general handling of external links has been to open them in a new tab.

Reply:
I checked it is working fine